### PR TITLE
Added Lore Balance Activity

### PR DESCRIPTION
**Pull request by Hornsoup**
Also reordered activities so they were a little more relative to each other and lore grouped.

### CODE REVIEW

File: src/app/game-state/character.ts [hunk 1]
```diff
@@ -426,6 +426,16 @@ export class Character {
     return increaseAmount;
   }
 
+  decreaseAttribute(attribute: AttributeType, amount: number): number {
```
Comment (immortalityidle) [inline src/app/game-state/character.ts:429]: This didn't end up being used, right?

File: src/app/game-state/activity.service.ts [hunk 2]
```diff
@@ -486,29 +486,32 @@ export class ActivityService {
...
+    newList.push(this.Alchemy);
     newList.push(this.BodyCultivation);
     newList.push(this.MindCultivation);
-    newList.push(this.CoreCultivation);
-    newList.push(this.InfuseEquipment);
-    newList.push(this.InfuseBody);
-    newList.push(this.ExtendLife);
+    if(this.characterService.characterState.manaUnlocked || this.characterService.characterState.easyMode){
+      newList.push(this.ManaControl);
+      newList.push(this.CoreCultivation);
```
Comment (immortalityidle) [inline src/app/game-state/activity.service.ts:508]: Can we just put ManaControl in the if?

File: src/app/game-state/character.ts (hunk 1 above)
Comment (Hornsoup) [reply src/app/game-state/character.ts:429]: Ah, forgot about that method.

**Comment by Hornsoup** [comment]
Took care of review points.

**Review by immortalityidle — APPROVED** [review]

**Pull request merged**